.github/workflows/ci.yml

```diff
@@ -144,6 +145,14 @@ jobs:
           cargo test --target wasm32-unknown-unknown --all-targets --all-features
           cargo test --target wasm32-unknown-unknown --all-targets --no-default-features
 
+      - name: Install Fuel Core
+        if: ${{ matrix.command == 'test' }}
+        run: |
+          curl -sSLf https://github.com/FuelLabs/fuel-core/releases/download/v${{ env.FUEL_CORE_VERSION }}/fuel-core-${{ env.FUEL_CORE_VERSION }}-x86_64-unknown-linux-gnu.tar.gz -L -o fuel-core.tar.gz
+          tar -xvf fuel-core.tar.gz          
+          chmod +x fuel-core-${{ env.FUEL_CORE_VERSION }}-x86_64-unknown-linux-gnu/fuel-core
+          mv fuel-core-${{ env.FUEL_CORE_VERSION }}-x86_64-unknown-linux-gnu/fuel-core /usr/local/bin/fuel-core
+
```

Comment:
What's the current situation with regards to this? Are we now effectively _using_ `fuel-core` as a binary and running our tests against it?

Reply:
Yes! In this way, we download the defined release of fuel-core bin. from `https://github.com/FuelLabs/fuel-core/releases` and then we use it to run the tests.